beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -92,3 +79,31 @@ func (s *Server) GetAggregateAttestation(w http.ResponseWriter, r *http.Request)
 		}}
 	network.WriteJson(w, response)
 }
+
+// SubmitContributionAndProofs publishes multiple signed sync committee contribution and proofs.
+func (s *Server) SubmitContributionAndProofs(w http.ResponseWriter, r *http.Request) {
+	var req SubmitContributionAndProofsRequest
+	if r.Body != http.NoBody {
+		if err := json.NewDecoder(r.Body).Decode(&req.Data); err != nil {
+			shared.HandleHTTPError(w, "Could not decode request body: "+err.Error(), http.StatusBadRequest)
+			return
+		}
+	}
+
+	validate := validator.New()
+	for _, item := range req.Data {
+		if err := validate.Struct(item); err != nil {
+			shared.HandleHTTPError(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+		consensusItem, err := item.ToConsensus()
+		if err != nil {
+			shared.HandleHTTPError(w, "Could not convert request contribution to consensus contribution: "+err.Error(), http.StatusBadRequest)
+			return
+		}
+		rpcError := core.SubmitSignedContributionAndProof(r.Context(), consensusItem, s.Broadcaster, s.SyncCommitteePool, s.OperationNotifier)
```

Comment:
is allowing for partial submission correct? ( if the data is a range this could fail half way and submit some) just wanted to make sure.

Reply:
each object is independent, so I think it's ok